### PR TITLE
Make timeline responsive to resize

### PR DESCRIPTION
**Pull request by lchan131**
Demo:

https://user-images.githubusercontent.com/98043376/161116476-741019a9-a51a-4e8a-8963-4774eccbe770.mov

### CODE REVIEW

File: capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js [hunk 1]
```diff
@@ -1037,6 +1087,23 @@ class CaptureProducerTimeline extends RtlMixin(InternalLocalizeMixin(LitElement)
 		this.timeline.getMarksOnTimeline().forEach(mark => mark.displayObject.mouseEnabled = this._controlMode === constants.CONTROL_MODES.MARK);
 	}
 
+	_updateRelativeWidth(percentage) {
+		const rootContainer = this.shadowRoot.querySelector('.d2l-video-producer-timeline');
+		const controlButtonWidth = 156;
```
Comment (qwangd2l) [inline capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js:1092]: What's control button width? Will it always be 156?

File: capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js [hunk 2]
```diff
@@ -18,9 +18,10 @@ class CaptureProducerTimeline extends RtlMixin(InternalLocalizeMixin(LitElement)
 			mediaPlayerEnded: { type: Boolean },
 			metadata: { type: Object },
 			timelineVisible: { type: Boolean },
-			width: { type: Number },
+			widthString: { type: String, attribute: 'width' },
 			videoLoaded: { type: Boolean },
 
+			width: { type: Number, attribute: false },
```
Comment (MichaelKovalchuk) [inline capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js:24]: if it is no longer an attribute, can we make it private? e.g. `_widthPixels`  and keep `widthString` as `width`?

File: capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js (hunk 1 above)
Comment (MichaelKovalchuk) [inline capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js:1092]: can we get control button width from the control button dom element?

Comment (lchan131) [reply capture/d2l-capture-producer/src/d2l-capture-producer-timeline.js]: Getting this value from the DOM instead

**Pull request merged**